asllib/doc/RelationsOnTypes.tex

```diff
@@ -2742,6 +2743,37 @@ \subsubsection{Formally}
   \getbitvectorwidth(\tenv, \vt) \typearrow \TypeErrorVal{\ExpectedBitvectorType}
 }
 \end{mathpar}
+\CodeSubsection{\GetBitvectorWidthBegin}{\GetBitvectorWidthEnd}{../Typing.ml}
+
+\subsubsection{TypingRule.GetBitvectorConstWidth}
+\hypertarget{def-getbitvectorconstwidth}{}
+The function
+\[
+  \getbitvectorconstwidth(\overname{\staticenvs}{\tenv} \aslsep \overname{\ty}{\vt}) \aslto
+  \overname{\N}{\vw} \cup \overname{\TTypeError}{\TypeErrorConfig}
+\]
+returns the natural number $\vw$, which represents the width of the bitvector type $\vt$
+in the static environment $\tenv$.
+\ProseOtherwiseTypeError
+
+\subsubsection{Prose}
+One of the following applies:
+All of the following apply:
```

Comment:
is there a level of itemize missing here? otherwise why have both enumeration preambles (One of the following applies / All of the following apply)?